scripts/azure_deployment/arm_aci.py

```diff
@@ -205,77 +205,79 @@ def make_aci_deployment(parser: ArgumentParser) -> Deployment:
         "resources": [],
     }
 
-    if not args.attestation_container_e2e:
-        deployment_name = args.deployment_name
-        container_name = args.deployment_name
-        container_image = args.aci_image
-        command = make_dev_container_command(args)
-        with_volume = args.aci_file_share_name is not None
-        containers = [
-            make_dev_container(
-                i, container_name, container_image, command, args.ports, with_volume
-            )
-            for i in range(args.count)
-        ]
-    else:
-        container_image = f"attestationcontainerregistry.azurecr.io/attestation-container:{args.deployment_name}"
-        deployment_name = f"{args.deployment_name}-business-logic"
-        container_name = f"{args.deployment_name}-attestation-container"
-        command = make_attestation_container_command(args)
-        args.ports.append(ATTESTATION_CONTAINER_PORT)
-        with_volume = False
-        containers = [
-            make_attestation_container(
-                container_name, container_image, command, args.ports
-            )
-        ]
-
-    container_group_properties = {
-        "sku": "Standard",
-        "containers": containers,
-        "initContainers": [],
-        "restartPolicy": "Never",
-        "ipAddress": {
-            "ports": [{"protocol": "TCP", "port": p} for p in args.ports],
-            "type": "Public",
-        },
-        "osType": "Linux",
-    }
-
-    container_group = {
-        "type": "Microsoft.ContainerInstance/containerGroups",
-        "apiVersion": "2022-04-01-preview",
-        "name": deployment_name,
-        "location": args.region,
-        "properties": container_group_properties,
-    }
-
-    if args.aci_file_share_name is not None:
-        container_group_properties["volumes"] = [
-            {
-                "name": "ccfcivolume",
-                "azureFile": {
-                    "shareName": args.aci_file_share_name,
-                    "storageAccountName": args.aci_file_share_account_name,
-                    "storageAccountKey": args.aci_storage_account_key,
-                },
-            }
-        ]
-
-    if not args.non_confidential:
-        if args.security_policy_file is not None:
-            with open(args.security_policy_file, "r") as f:
-                security_policy = f.read()
+    for i in range(args.count):
+
+        if not args.attestation_container_e2e:
+            deployment_name = args.deployment_name
+            container_name = args.deployment_name
+            container_image = args.aci_image
+            command = make_dev_container_command(args)
+            with_volume = args.aci_file_share_name is not None
+            containers = [
+                make_dev_container(
+                    i, container_name, container_image, command, args.ports, with_volume
+                )
+                for i in range(args.count)
```

Comment:
This should go then?

Reply:
Whoops yeah that's a merge conflict typo, good spot

Reply:
https://github.com/microsoft/CCF/pull/4854/commits/ff816e90f624ca165b297fe1ec3312ea9057e3db